### DISCUSSION TITLE
Rename FileSystem to Files

### DISCUSSION THREAD

**Discussion by passivestar** [godotengine/godot-proposals#9071] [Editor]
I asked around and people liked this, so opening a discussion

FileSystem stands out from what the other docks are called, it's unexpectedly more technical than it needs to be, and it's longer than other docks' names because it consists of two words unlike all of the others. Compare to the more concise "Scene", "Inspector", "Node", etc

Operating systems have moved on from this too and when two regular people are talking about files on disk you'd usually hear "file system" when there's a conversation about the types of file systems (NTFS, FAT, APFS, etc)

This is a small change that I believe makes a noticable difference for editor's user-friendliness by reducing technical jargon in its UI

![fs](https://github.com/godotengine/godot-proposals/assets/60579014/617bb75a-e109-4c10-9f2b-0189c9bacebe)

**Comment by AThousandShips** [comment]
Thank you for your proposal, closing as a duplicate of:
* https://github.com/godotengine/godot-proposals/discussions/9067

See there for various details

**Reply by passivestar** [reply]
what a coincidence, ok thanks 😄
